packages/tldraw/src/lib/ui/hooks/useCommandBarActions.ts

```diff
@@ -0,0 +1,32 @@
+import { unwrapLabel, useActions } from '../context/actions'
+import { TLUiTranslationKey } from './useTranslation/TLUiTranslationKey'
+import { useTranslation } from './useTranslation/useTranslation'
+
+/** @public */
+export function useCommandBarActions(search: string) {
+	const actions = useActions()
+	const msg = useTranslation()
+	return Object.values(actions)
+		.filter((action) => {
+			// Don't show actions until we start searching
+			if (search === '') return false
+			// If we didn't want to show the disabled actions
+			// if (!action.enabled?.()) return false
+			const unwrapped = unwrapLabel(action.label, 'command-bar')
+			const value = msg(unwrapped as TLUiTranslationKey)
+			if (!value) return false
+			// Alternative to the search check above we can show all actions if we haven't started typing yet
+			// if (search === '') return true
+			return value.toLowerCase().includes(search.toLowerCase())
```

Comment:
should we do a fuzzy search or this substring search?

Reply:
Yeah, I explored this a bit. Most fuzzy search libraries are quite big, but I guess we could go with something really simple.

Reply:
yeah that sounds good. something for a future iteration sounds like a good plan 👍